index.js

```diff
@@ -7,7 +7,6 @@ const dialogOpts = {
   title: 'Headset Update',
   message: 'New Version is Available for Headset!',
   detail: 'Restart the application to apply the updates.',
-  icon: './icon.png',
```

Comment:
Do you want to try and use an absolute path instead?
```js
path.join(__dirname, 'icon.png'),
```

Reply:
Also, we could do this: https://github.com/electron/electron/issues/7657#issuecomment-255673246

Reply:
Yes. I'll do it this evening

Reply:
The more I read, the more I realized I should have used `nativeImage`, I don't think a dialog can take a string for the icon. [`dialog.showMessageBox`](https://electronjs.org/docs/api/dialog#dialogshowmessageboxbrowserwindow-options-callback) can only take `nativeImage` and not strings